addons/project/static/src/views/project_project_list/project_project_list_controller.js

```diff
@@ -0,0 +1,21 @@
+import { ListController } from "@web/views/list/list_controller";
+import { onWillStart } from "@odoo/owl";
+import { user } from "@web/core/user";
+
+export class ProjectListController extends ListController {
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.isProjectManager = await user.hasGroup('project.group_project_manager');
+        });
+    }
+
+    getStaticActionMenuItems() {
+        const actionMenuItems = super.getStaticActionMenuItems(...arguments);
+        const menuItems = ['duplicate', 'archive', 'unarchive'];
+        if (!this.isProjectManager) {
+            menuItems.forEach(item => delete actionMenuItems[item]);
+        }
+        return actionMenuItems;
```

Comment:
Instead of removing, we should not add menu items, and why only for the list view?

Reply:
Changes done :+1: